pack/fhv/fhvc.json

```diff
@@ -283,6 +283,178 @@
     "traits": "Blight. Colour.",
     "type_code": "skill"
   },
+  {
+    "back_name": "Bertie's Choice",
+    "back_text": "<b>Check the Campaign Log. If <i>Bertie had an epiphany</i>:\n<blockquote><i>You reconvene with Dr. Marquez at the Boarding House. Bertie approaches you with a manic gleam in his eye, wearing the white-gold cloak of the Vale. \"I've decided to join the Feast today - for the whole day. I understand this may put my grant in jeopardy but -\" Before Dr. Marquez can chastise him, the young man scampers off past a throng of dancers.</i></blockquote>\n<b>(→R2)</b> <i>(page 54)</i>.\n<hr><b>Otherwise:</b>\n<blockquote><i>You reconvene with Dr. Marquez at the Boarding House. Bertie approaches you and the professor with obvious excitement. \"I've had a revelation about Hemlock Isle. The food chain is chaos. Predators predated by prey - poetic, isn't it? Something in the food or water induces a frenzy - in fact, this entire celebration is kind of frenzy,\" he gestures to the giddy figures and dancing children.</i></blockquote>\n<b>(→R1)</b> <i>(page 54)</i>.",
```

Comment:
```suggestion
    "back_text": "<b>Check the Campaign Log. If <i>Bertie had an epiphany</i>:</b>\n<blockquote><i>You reconvene with Dr. Marquez at the Boarding House. Bertie approaches you with a manic gleam in his eye, wearing the white-gold cloak of the Vale. \"I've decided to join the Feast today - for the whole day. I understand this may put my grant in jeopardy but -\" Before Dr. Marquez can chastise him, the young man scampers off past a throng of dancers.</i></blockquote>\n<b>(→R2)</b> <i>(page 54)</i>.\n<hr><b>Otherwise:</b>\n<blockquote><i>You reconvene with Dr. Marquez at the Boarding House. Bertie approaches you and the professor with obvious excitement. \"I've had a revelation about Hemlock Isle. The food chain is chaos. Predators predated by prey - poetic, isn't it? Something in the food or water induces a frenzy - in fact, this entire celebration is kind of frenzy,\" he gestures to the giddy figures and dancing children.</i></blockquote>\n<b>(→R1)</b> <i>(page 54)</i>.",

```